pkg/arvo/app/publish.hoon

```diff
@@ -196,7 +196,7 @@
           =/  book=notebook-info
             [title.old '' =(%open comments.old) / /]
           =+  ^-  [grp-car=(list card) write-pax=path read-pax=path]
-            (make-groups book-name group-pax ~ %.n %.n)
```

Comment:
What's this change about?

Reply:
Another publish OTA issue that I encountered before the ford issue. I can make it its own PR if you prefer.

Reply:
I'm ok with it being in this PR, but I don't know how to review it.  Was it just a namespace type where +make-groups really needs to be +make-groups:main, or was it something else?

Reply:
Yes thats exactly it. Without `:main` the call to make-groups had a bunted bowl, which caused problems when it tried to use the current date in a particular way.

Reply:
Ok